test/test_quantized.py

```diff
@@ -252,6 +252,44 @@ def test_qclamp(self, X, min_val, max_val):
             qY_hat = op(qX, min_val, max_val)
             self.assertEqual(qY, qY_hat, message="{} qclamp failed".format(name))
 
+    """Tests the correctness of the quantized::hardtanh op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 8, 1, 8),
+                       elements=hu.floats(-1e6, 1e6, allow_nan=False),
+                       qparams=hu.qparams()),
+           min_val=hu.floats(-1e6, 1e6, allow_nan=False),
```

Comment:
`allow_infinity=False` See [documentation](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.floats) for more details